app/src/main/java/org/schabi/newpipe/ui/components/common/ServiceColoredButton.kt

```diff
@@ -0,0 +1,54 @@
+package org.schabi.newpipe.ui.components.common
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.schabi.newpipe.ui.theme.AppTheme
+import org.schabi.newpipe.ui.theme.SizeTokens.SpacingMedium
+import org.schabi.newpipe.ui.theme.SizeTokens.SpacingSmall
+
+@Composable
+fun ServiceColoredButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
```

Comment:
Please place modifiers before any other parameters in composable. Lint shows a warning otherwise.